packages/template-blank-svelte/.gitignore

```diff
@@ -0,0 +1,16 @@
+npm-debug.log
+.DS_Store
+
+*.js.map
+*.js
```

Comment:
Not sure about including *.js here.
The current svelte template doesn't have it, and neither does the vue one https://github.com/NativeScript/nativescript-app-templates/blob/master/packages/template-blank-vue/.gitignore

Reply:
I just noticed that this may have bitten us already. The .gitignore file ignores svelte.config.js which is assumed to exist by the webpack plugin https://github.com/NativeScript/NativeScript/pull/8963/files#diff-34201858e4b5b4e6eaad695e47ee0ec6b6ae4b60ccc81bab535438a0520895e8R15

Reply:
@halfnelson this gitignore is actually for the templates repo - the one that's used/ends up in projects is from `packages/template-blank-svelte/tools/dot.gitignore`